sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/aggregator/AggregatorHandle.java

```diff
@@ -105,6 +112,7 @@ public final void recordLong(long value) {
 
   @Override
   public final void recordLong(long value, Attributes attributes, Context context) {
```

Comment:
I think this method should move up one in the class so it calls "down" to the other `recordLong` which then calls `doRecordLong`.  Same for the double method ordering.

Reply:
Right, eventually this gets refactored/shrunk.  I've been trying to limit the size of these CLs so far, so all that extra gunk is still there (but not needed).